versions-maven-plugin/src/it/it-changerecord-update-parent-001/verify.groovy

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
eh... once again it is not AFS project - new files should have not such headers

Please check other new files also.

Reply:
Oops. Will do.

Reply:
By the way, what do you think of introducing the rat plugin?

Reply:
Edit: I've just checked, there's really **plenty** of files with the old license. Will create a separate PR to update those.

Reply:
@slawekjaranowski phew, updated the changed files only. For the rest a separate PR should be created.